src/lib/webcontainer.ts

```diff
@@ -832,17 +927,22 @@ async function get_tree(
 					contents,
 				},
 			};
-		} else if (node.isDirectory() && !IGNORE_LIST.includes(node.name)) {
+		} else if (
+			node.isDirectory() &&
+			(!IGNORE_LIST.includes(node.name) ||
+				(ignore_node_modules === false && node.name === 'node_modules'))
+		) {
 			tree[node.name] = {
 				directory: await get_tree(
 					await webcontainer_instance.fs.readdir(node_path, { withFileTypes: true }),
 					node_path + '/',
 					as_string,
+					true,
 				),
 			};
 		}
 	}
 	return tree;
 }
 
-const IGNORE_LIST = ['.svelte-kit', 'node_modules'];
+const IGNORE_LIST = ['.svelte-kit/generated', 'node_modules'];
```

Comment:
was this for debugging purposes, or do we need this? i think it makes the .svelte-kit folder appear in the file tree which we might not want